.circleci/config.yml

```diff
@@ -137,14 +137,15 @@ jobs:
             echo "Running $pkgs"
             for pkg in $pkgs
             do
-              if ! gotestsum --no-summary=all --jsonfile=jsonfile-${pkg////-} -- $pkg -p 1 -timeout 30m -failfast \
+              if ! gotestsum --no-summary=all --jsonfile=jsonfile-${pkg////-} -- $pkg -p 1 -timeout 50m -failfast \
                     -use-kind \
-                    -enable-multi-cluster \
                     -enable-enterprise \
+                    -enable-multi-cluster \
                     -kubecontext="kind-dc1" \
                     -secondary-kubecontext="kind-dc2" \
                     -debug-directory="$TEST_RESULTS/debug" \
-                    -consul-k8s-image=docker.mirror.hashicorp.services/hashicorpdev/consul-k8s:latest
+                    -consul-image="ishustava/consul-enterprise:tproxy-test" \
+                    -consul-k8s-image="ishustava/consul-k8s-dev:04-13-2021-8f91d97" # TODO: change once feature-tproxy consul-k8s branch is merged
```

Comment:
Reminder to change the consul and consul-k8s images. Will we do that as part of this PR or a future one?

Reply:
Good call, I'll add it to our release checklist. I think we should update it once both images are released and before we release the helm chart. Another reason we need to do it is to test that the released consul-k8s image works with the iptables command now included.